README.md

```diff
@@ -66,6 +66,35 @@ in progress.
 LNURL-auth.
 VSS is also integrated with [LDK-node] v0.4.x as alpha support.
 
+### Development
+
+* **Build & Deploy**: Refer to language specific folder for instructions related to build and deploy of VSS.
+* **Hosting**: VSS can either be self-hosted or deployed in the cloud. If a service provider is hosting VSS for multiple
+  users, it must be configured with **HTTPS**, **Authentication/Authorization**, and **rate-limiting**.
+* **Authentication and Authorization**: Currently, the VSS-server
+  supports [JWT](https://datatracker.ietf.org/doc/html/rfc7519)-based authentication and authorization, and can run
+  without authentication for local testing or in trusted setups. The VSS-rust-client supports LNURL-auth & JWT based
+  authentication and authorization. Switching to simple HTTP header authentication is straightforward by adding another
+  implementation. Note that the security of authentication heavily relies on using HTTPS for all requests.
+* **Scaling**: VSS itself is stateless and can be horizontally scaled easily. VSS can be configured to point to a
+  PostgreSQL cluster, and further scaling considerations need to be addressed in the PostgreSQL cluster.
+* **Using with LDK-node**: [LDK-node] can be easily configured to run with VSS as primary storage, and there is also
+  support of other languages such as [Swift][swift], [Kotlin][kotlin], and [Python][python].
```

Comment:
The second part of the compound sentence should be an independent sentence, IMO. Also, there is no reference to rust in the first part, so the talk if different languages seems out of place. Consider re-wording or moving the second part after the example.